test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -626,8 +616,8 @@ public Task ResponseStatusCodeSetBeforeHttpContextDisposedRequestMalformedReadIg
 
             var logMessage = Assert.Single(testLogger.Messages, message => message.LogLevel == LogLevel.Error);
             Assert.Equal(
-                $"Response Content-Length mismatch: too many bytes written (12 of 5).",
-                logMessage.Exception.Message);
+                "Response Content-Length mismatch: too many bytes written (12 of 5).",
+                logMessage?.Exception.Message);
```

Comment:
The `?` is unnecessary since the previous assert will fail if it can't find a log message.